pkg/ddl/column.go

```diff
@@ -1820,8 +1820,7 @@ func updateColumnDefaultValue(d *ddlCtx, t *meta.Meta, job *model.Job, newCol *m
 		oldCol.AddFlag(mysql.NoDefaultValueFlag)
 	} else {
 		oldCol.DelFlag(mysql.NoDefaultValueFlag)
-		sctx := newReorgSessCtx(d.store)
```

Comment:
Do we need to change to other places where we are calling `newReorgSessCtx`? like `buildDescTableScan` in reorg.go

Reply:
It's in another PR: https://github.com/pingcap/tidb/pull/53467 .